docs/source/notes/get_start_xpu.rst

```diff
@@ -0,0 +1,339 @@
+Pytorch 2.4: Getting Started on Intel GPU
+=========================================
+
+The support for Intel GPUs is released alongside PyTorch v2.4.
+
+This release only supports build from source for Intel GPUs.
+
+Hardware Prerequisites
+----------------------
+
+.. list-table::
+   :header-rows: 1
+
+   * - Supported Hardware
+     - Intel® Data Center GPU Max Series
+   * - Supported OS
+     - Linux
+
+
+PyTorch for Intel GPUs is compatible with Intel® Data Center GPU Max Series and only supports OS Linux with release 2.4.
+
+Software Prerequisites
+----------------------
+
+As a prerequisite, install the driver and required packages by following the `PyTorch Installation Prerequisites for Intel GPUs <https://www.intel.com/content/www/us/en/developer/articles/tool/pytorch-prerequisites-for-intel-gpus.html>`_.
+
+Set up Environment
+------------------
+
+Before you begin, you need to set up the environment. This can be done by sourcing the ``setvars.sh`` script provided by the ``intel-for-pytorch-gpu-dev`` and  ``intel-pti-dev`` packages.
+
+.. code-block::
+
+   source ${ONEAPI_ROOT}/setvars.sh
+
+.. note::
+   The ``ONEAPI_ROOT`` is the folder you installed your ``intel-for-pytorch-gpu-dev`` and  ``intel-pti-dev`` packages. Typically, it is located at ``/opt/intel/oneapi/`` or ``~/intel/oneapi/``.
+
+Build from source
+-----------------
+
+Now we have all the required packages installed and environment acitvated. Use the following commands to install ``pytorch``, ``torchvision``, ``torchaudio`` by building from source. For more details, refer to official guides in `PyTorch from source <https://github.com/pytorch/pytorch?tab=readme-ov-file#intel-gpu-support>`_, `Vision from source <https://github.com/pytorch/vision/blob/main/CONTRIBUTING.md#development-installation>`_ and `Audio from source <https://pytorch.org/audio/main/build.linux.html>`_.
+
+.. code-block::
+
+   # Get PyTorch Source Code
+   git clone --recursive https://github.com/pytorch/pytorch
+   cd pytorch
+   git checkout main # or checkout the specific release version >= v2.4
+   git submodule sync
+   git submodule update --init --recursive
+
+   # Get required packages for compilation
+   conda install cmake ninja
+   pip install -r requirements.txt
+
+   # Pytorch for Intel GPUs only support Linux platform for now.
+   # Install the required packages for pytorch compilation
```

Comment:
I suggest the following change to address my concerns below. This would highlight few things:
1. That these packages (`intel::mkl-static intel::mkl-include`) are required for Intel CPUs optimal performance
2. That these packages are different packages vs. those which might seem the same packages installed via `sudo apt install intel-for-pytorch-gpu-dev intel-pti-dev`
3. That these packages are in a way optional, pytorch might build w/o them, but suffice suboptimal performance on Intel CPUs
```suggestion
   # To achieve optimal performance on Intel CPUs, additionally install the following packages
```

Reply:
`# Install the required packages for pytorch compilation for Intel CPU optimizations`  How's this change look?

Reply:
Hi @dvrogozh @jingxu10 , since it is a required package and also to mention it is for Intel CPU optimizations, I changed the comment to 
`Install the required packages for pytorch compilation with Intel CPU optimizations`  and marked this as resolved.

Reply:
Hi @dvrogozh , thanks a lot for your suggestions!
As we discussed, the mkl packages is a necessary package for CPU optimization, so we'd better leave it as a required packages.

Reply:
From my opinion the comment gives misleading impression to the user and needs to change. I am not ok with this discussion marked as resolved and do not consider it as such.

Reply:
Hi @dvrogozh , what do you think we keep the required packages: `Install the required packages for pytorch compilation with Intel CPU optimizations.`